client/src/components/Leaflet.tsx

```diff
@@ -15,8 +15,12 @@ import { MarkerClusterGroup } from "leaflet.markercluster"
 import "leaflet.markercluster/dist/MarkerCluster.css"
 import "leaflet.markercluster/dist/MarkerCluster.Default.css"
 import "leaflet/dist/leaflet.css"
+import { gql } from "@apollo/client"
+import API from "api"
+import LinkTo from "components/LinkTo"
```

Comment:
These new imports should come at the top (but apparently the plugin that should do the automatic sorting [@ianvs/prettier-plugin-sort-imports] does not pick this up…).